libyul/backends/evm/ControlFlowGraphBuilder.cpp

```diff
@@ -517,14 +515,16 @@ Stack const& ControlFlowGraphBuilder::visitFunctionCall(FunctionCall const& _cal
 	yulAssert(m_scope, "");
 	yulAssert(m_currentBlock, "");
 
+	Stack const* output = nullptr;
+	bool canContinue = true;
```

Comment:
Correct here? Shadowing `ControlFlowGraphBuilder::canContinue`?

Reply:
Not sure I follow :-). I don't see ``ControlFlowGraphBuilder::canContinue``...

Reply:
Hmm.. not sure what I've seen here.. I don't see anything strange now...